ksqldb-engine/src/main/java/io/confluent/ksql/util/ExecutorUtil.java

```diff
@@ -73,9 +91,7 @@ public static void executeWithRetries(
         return executable.call();
       } catch (final Exception e) {
         final Throwable cause = e instanceof ExecutionException ? e.getCause() : e;
-        if (cause instanceof RetriableException
-            || (cause instanceof Exception && retryBehaviour == RetryBehaviour.ALWAYS)) {
-          log.info("Retrying request. Retry no: " + retries, e);
```

Comment:
Was this log message deleted intentionally?

Reply:
ah good catch!